### PR TITLE
optimize chainstore call function,fix operator map panic

### PR DESCRIPTION
**Pull request by xiemylogos**

### CODE REVIEW

File: consensus/vbft/block_pool.go [hunk 1]
```diff
@@ -755,3 +756,27 @@ func (pool *BlockPool) onBlockSealed(blockNum uint32) {
...
+	return pool.chainStore.getExecWriteSet(blkNum)
+}
+
+func (pool *BlockPool) submitBlock(blkNum uint32) error {
+	pool.lock.RLock()
+	defer pool.lock.RUnlock()
+	return pool.chainStore.submitBlock(blkNum)
+}
+
+func (pool *BlockPool) getBlock(blockNum uint32) (*Block, error) {
```
Comment (Honglei-Cong) [inline consensus/vbft/block_pool.go]: block pool has getSealedBlock, what's their difference?

File: consensus/vbft/block_pool.go [hunk 2]
```diff
@@ -755,3 +756,27 @@ func (pool *BlockPool) onBlockSealed(blockNum uint32) {
...
+}
+
+func (pool *BlockPool) getExecWriteSet(blkNum uint32) *overlaydb.MemDB {
+	pool.lock.RLock()
+	defer pool.lock.RUnlock()
+	return pool.chainStore.getExecWriteSet(blkNum)
+}
+
+func (pool *BlockPool) submitBlock(blkNum uint32) error {
+	pool.lock.RLock()
```
Comment (Honglei-Cong) [inline consensus/vbft/block_pool.go]: pool.lock.Lock() ?

File: consensus/vbft/chain_store.go [hunk 3]
```diff
@@ -133,15 +133,20 @@ func (self *ChainStore) AddBlock(block *Block) error {
 		return fmt.Errorf("chainstore AddBlock GetBlockExecResult: %s", err)
 	}
 	self.pendingBlocks[blkNum] = &PendingBlock{block: block, execResult: &execResult, hasSubmitted: false}
-	self.pid.Tell(
-		&message.BlockConsensusComplete{
-			Block: block.Block,
-		})
+	if _, present := self.pendingBlocks[blkNum-2]; present {
```
Comment (Honglei-Cong) [inline consensus/vbft/chain_store.go]: If need clean here, loop all pendings to clean unused entries

File: consensus/vbft/block_pool.go (hunk 1 above)
Comment (xiemylogos) [reply consensus/vbft/block_pool.go]: no difference.I will del this function later.

File: consensus/vbft/block_pool.go (hunk 2 above)
Comment (xiemylogos) [reply consensus/vbft/block_pool.go]: yes,I will recovery delete pendingblock entry operator to submit function in chainstore

File: consensus/vbft/chain_store.go (hunk 3 above)
Comment (xiemylogos) [reply consensus/vbft/chain_store.go]: recover this delete pendingblock entry  to sumit function.

**Review by Honglei-Cong — DISMISSED** [review]

**Review by Honglei-Cong — APPROVED** [review]

**Pull request merged**